### PR TITLE
326 Popup fetches data about modal split for a CT

### PR DESCRIPTION
**Pull request by allthesignals**
This PR adds some new components to the transportation map hover which lookup data.

Because we're blocked by lack for real data for these splits, these changes setup Mirage mocks, which make several assumptions about implementation of #274.

This PR also changes some internals of existing components.

687474703a2f2f7265636f726469742e636f2f6d70306c7537686b644e2e676966

### CODE REVIEW

**Comment by allthesignals** [comment]
@julialucyhogan deploy preview failed 🤔

File: frontend/app/templates/components/transportation/study-area-map/census-tract-data.hbs [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{{#throttle-property property=geoid as |geoid|}}
```
Comment (godfreyyeung) [inline frontend/app/templates/components/transportation/study-area-map/census-tract-data.hbs]: I like that it only throttles data loading, and not the popup itself. Also that this is expressed by the template!

File: frontend/app/templates/components/mapbox/current-mouse-position.hbs [hunk 2]
```diff
@@ -1,4 +1,6 @@
 {{!-- binds a map event with an action using mapbox-gl's on method --}}
-{{mapbox-gl-on this.mapboxEventName (action 'handleEvent') eventSource=map.instance}}
+{{mapbox-gl-on 'mousemove' (action 'handleMouseMove') eventSource=map.instance}}
```
Comment (godfreyyeung) [inline frontend/app/templates/components/mapbox/current-mouse-position.hbs]: heads up @julialucyhogan  and I also headed in the direction of generalizing current-mouse-position. We made a `current-mouse-event` component as a part of #279 . Later when we merge in #279 we may need to reconcile all the different patterns

Comment (allthesignals) [reply frontend/app/templates/components/mapbox/current-mouse-position.hbs]: @godfreyyeung cool, sounds like we could use your more generic approach. let's reconcile later. thanks

**Pull request merged**